rpc_util_test.go

```diff
@@ -173,8 +174,8 @@ func TestContextErr(t *testing.T) {
 		// outputs
 		errOut transport.StreamError
 	}{
-		{context.DeadlineExceeded, transport.StreamErrorf(codes.DeadlineExceeded, "%v", context.DeadlineExceeded)},
-		{context.Canceled, transport.StreamErrorf(codes.Canceled, "%v", context.Canceled)},
+		{context.DeadlineExceeded, transport.StreamError{codes.DeadlineExceeded, fmt.Sprintf("%v", context.DeadlineExceeded)}},
```

Comment:
not too important, but you can use `.Error()` instead of `fmt.Sprintf("%v", ...)` here and on the next line